acrobat/blocks/eventwrapper/eventwrapper.css

```diff
@@ -50,13 +50,27 @@ body:not([data-current-event]) [data-event-name=onload] {
   animation-name: fade-in;
 }
 
-
-@keyframes fade-in { 
-  0% { 
+@keyframes fade-in {
+  0% {
     opacity: 0;
   }
 
-  100% { 
+  100% {
     opacity: 1;
   }
 }
+
+@media (max-width: 600px) {
+  .wapper-rotate-pages.fsw > div > div {
+    height: auto;
+  }
+}
+
+.wapper-rotate-pages .LifecycleOrganizeSettings__organizeSettings___2EChZ {
```

Comment:
I don't think it's safe to reference webpack-generated hash values from our build.   Are these style changes that we should be making in the widget?

Reply:
@draganatrajkovic if this is the best solution we can do something like this 
`div[id*="LifecycleOrganizeSettings__organizeSettings"]`
assuming it's a `div` :)

Reply:
@Blainegunn Thank you :)
@peller I fixed the code and small bugs, so please take a look.